systemd/user/adsys-user-mounts.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=ADSys user mount handler
+After=network-online.service
+ConditionPathExists=/run/adsys/users/%U/mounts
+ConditionPathExists=/usr/libexec/adsys_mount
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
```

Comment:
Tbh, I have no idea if that's precisely what we want. This would trigger the binary to run again after a policy refresh, which could lead to lots of "location is already mounted" warns so maybe it's better to lock it to a `one time per session` strategy?
I'm not sure about it, btw... This is just to trigger some thinking and discussion about it.

Reply:
Actually, this is already what happens btw... We don't control this service from the adsys daemon. This is something that will be enabled when the adsys package is installed and then it's up to systemd, which means it will only happen at log in.